<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="YADHU" failures="1" tests="3" name="testRunerTest.MyRunner" time="37.811" errors="0" timestamp="2025-06-22T12:59:59 CEST" skipped="0">
  <testcase classname="testRunerTest.MyRunner" name="runScenario" time="11.747"/>
  <testcase classname="testRunerTest.MyRunner" name="runScenario" time="11.746">
    <failure message="May have a bug expected [No data] but found [Beetroot]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: May have a bug expected [No data] but found [Beetroot]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at stepDefFiles.OfferPageStepDef.check_if_product_exist(OfferPageStepDef.java:31)
at âœ½.Check if product exist(file:///C:/Users/yadhu/eclipse-workspace/Cucumber_RSA/src/test/resources/FeatureFiles/searchProduct.feature:10)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="testRunerTest.MyRunner" name="runScenario" time="14.318"/>
</testsuite> <!-- testRunerTest.MyRunner -->
